README.md

```diff
@@ -187,7 +187,11 @@ docker run -d -p 2801:2801 -e SLACK_WEBHOOKURL=XXXX -e DATADOG_APIKEY=XXXX falco
 
 * Download the latest release:
   ```shell
-  wget https://github.com/falcosecurity/falcosidekick/releases/latest -o falcosidekick
+  VER=$(curl --silent -qI https://github.com/falcosecurity/falcosidekick/releases/latest | awk -F '/' '/^location/ {print  substr($NF, 1, length($NF)-1)}')
+
+wget -c https://github.com/falcosecurity/falcosidekick/releases/download/${VER}/falcosidekick_${VER}_linux_arm64.tar.gz -O - | tar -xz
+  or
+wget -c https://github.com/falcosecurity/falcosidekick/releases/download/${VER}/falcosidekick_${VER}_linux_amd64.tar.gz -O - | tar -xz
```

Comment:
can you please adjust the alignment and also do a rebase and squash the commits?

thanks